client/model.go

```diff
@@ -43,3 +49,61 @@ func (client *Client) Remodel(b []byte) (changeID string, err error) {
 
 	return client.doAsync("POST", "/v2/model", nil, headers, bytes.NewReader(data))
 }
+
+// CurrentModelAssertion returns the current model assertion
+func (client *Client) CurrentModelAssertion() (*asserts.Model, error) {
+	assert, err := currentAssertion(client, "/v2/model")
+	if err != nil {
+		return nil, err
+	}
+	modelAssert, ok := assert.(*asserts.Model)
+	if !ok {
+		return nil, fmt.Errorf("invalid assertion type (%T) returned", assert)
```

Comment:
we know it's an assertion though, so we can use assert.Type().Name instead of %T, also probably s/invalid/unexpected/

Reply:
Done